src/App.js

```diff
@@ -15,6 +18,20 @@ import {
 } from "./screens";
 
 function App() {
+  const { videoDispatch } = useVideos();
+  let videoConfig = {
+    method: "get",
```

Comment:
You can move this video outside the component. Putting it inside will create new instance with each render

Reply:
Thanks, @Debarshi95  for suggesting this will implement this.